src/script/main/app.ts

```diff
@@ -914,7 +912,7 @@ class App {
 // Setting up the App
 //##############################################################################
 
-$(async () => {
+(async () => {
```

Comment:
Does this work? Cause it's not the equivalent of `$(...)`. 
If you tell me it works, (the app loads) then we are good